source/_page_templates/ability_template.rst

```diff
@@ -0,0 +1,58 @@
+.. [insert ability name here, lowercase, one word]:
+############
+ABILITY NAME
+############
+
+Description
+###########
+
+Paste description here. Insert gif here |ABILNAMEHERE1|
```

Comment:
In-game image or gif should probably go above or to the side of the description, not below.

Reply:
It is on the side?